<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Login
{
background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 rgb(226,238,243),
                                stop:1 rgb(240,207,214));
}
#title
{
font: 24pt &quot;楷体&quot;;
font-weight:bold;
background-color: rgba(202, 221, 245,0.5);
background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:1 rgb(226,238,243),
                                stop:0 rgb(240,207,214));
border-radius:20px;
}
#cp
{
font: 11pt &quot;楷体&quot;;
font-weight:bold;
}
#usernameLab
{
font: 18pt &quot;方正粗黑宋简体&quot;;
color: rgb(3, 57, 103);
}
#passwordLab
{
font: 18pt &quot;方正粗黑宋简体&quot;;
color: rgb(3, 57, 103);
}
QLineEdit
{
font: 14pt &quot;楷体&quot;;
font-weight:bold;
background-color: rgba(136, 193, 150,0.3);
border-radius:10px;
border:2px solid gray;
}
QLineEdit:hover
{
font: 14pt &quot;楷体&quot;;
font-weight:bold;
border:2px solid ;
}
#IdentityLab
{
font: 18pt &quot;方正粗黑宋简体&quot;;
color: rgb(3, 57, 103);
}
#btnLogin
{
font: 20pt &quot;楷体&quot;;
font-weight:bold;
border-radius:10px;
border:2px solid gray;
background-color: rgba(202, 221, 245,0.7);
}
#btnLogin:hover
{
font: 22pt &quot;楷体&quot;;
font-weight:bold;
border:3px solid;
background-color: rgb(202, 221, 245);
}</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>10</y>
     <width>740</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>740</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>740</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>青年大学习后台管理系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLab">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>490</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>490</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLab">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PassWord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>490</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdentityLab">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Identity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="IdentityBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>471</y>
     <width>490</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QCombobox主体*/
QComboBox {
	font: 14pt &quot;楷体&quot;;
	font-weight:bold;
	background-color: rgba(136, 193, 150,0.3);
	border-radius:10px;
	border:2px solid gray;
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
	padding-left: 20px;/*左侧边距*/
}
QComboBox:hover
{
	border:3px solid;
}
/*QCombobox右侧按钮*/
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}
/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/ptr/down.png);/*自定义图片填充*/
	width: 28px;/*设置该图标的宽高*/
	height: 28px;
}
/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow::on {
	border-image: url(:/ptr/up.png);/*自定义图片填充*/
	width: 28px;/*设置该图标的宽高*/
	height: 28px;
}


/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
	font: 16pt &quot;楷体&quot;;
	font-weight: bold;  
    border: 2px solid white;/*边框宽度、线形、颜色*/
	/*背景颜色*/
	background-color: rgba(136, 193, 150,0.3);
    border-radius: 10px;/*圆角*/
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    min-width: 10em;   /*# 组合框的最小宽度*/
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
	font: 16pt &quot;楷体&quot;;
	font-weight: bold;
	border-radius: 10px;/*圆角*/
    height: 50px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgba(136, 193, 150,0.3);

}

/*以下部分不知为何不生效，有待调试*/
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
	font: 16pt &quot;楷体&quot;;
	font-weight: bold;
       /* 整个下拉窗体越过每项的背景色 */
	background-color: rgba(136, 193, 150,0.5);
}

/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
	font: 16pt &quot;楷体&quot;;
	font-weight: bold;
	background-color: rgba(136, 193, 150,0.5);
}
</string>
   </property>
   <item>
    <property name="text">
     <string>团支书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老师</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="cp">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>670</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>出品:网络工程系党支部</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
